app/core/models/ocelot/package.scala

```diff
@@ -41,11 +40,12 @@ package object ocelot {
   val linkPattern: String = s"\\[(button|link)(-same|-tab)?:(.+?):($StanzaIdPattern|$HttpUriPattern)\\]"
   val timeConstantPattern: String = s"^($TenDigitIntPattern)\\s*(days?|weeks?|months?|years?)$$"
   val PositiveIntListPattern: String = s"^$TenDigitIntPattern(?:,$TenDigitIntPattern)*$$"
+  val DatePlaceHolderPattern: String = s"\\[date:($DatePattern|$LabelPattern)?:(year|month_num|month_start|month_end|month_name|dow_num|dow_name|day|)\\]"
```

Comment:
Regex has a trailing | after day

Also the dow and month functions are missing